src/pypromice/process/L2toL3.py

```diff
@@ -2,14 +2,20 @@
 """
 AWS Level 2 (L2) to Level 3 (L3) data processing
 """
+import pandas as pd
 import numpy as np
 import xarray as xr
+import toml, os
+from sklearn.linear_model import LinearRegression
 
-def toL3(L2, T_0=273.15, z_0=0.001, R_d=287.05, eps=0.622, es_0=6.1071, 
-         es_100=1013.246):
+import logging
+logger = logging.getLogger(__name__)
+
+def toL3(L2, config_folder='../aws-l0/metadata/station_configurations/', T_0=273.15):
```

Comment:
See this PR for inspiration https://github.com/GEUS-Glaciology-and-Climate/pypromice/pull/274

Reply:
It is fine - for now 😄 . 

I would also like to add functionalities that verifies that the format of the configuration file is correct. A TypedDict, a Dataclass or other explicit data structures makes it easier to maintain the code because the attributes are explicit. This will be in a later PR.